troubleshooting/debugging.md

```diff
@@ -169,6 +170,24 @@ The following are common network connection errors that we have seen frequently:
 [error] [tls] SSL error: NET - Connection was reset by peer
 ```
 
+#### CloudWatch Broken Connection Error
+
+As mentioned in the section, [Common Network Errors](#common-network-errors), seeing transinent network errors such as `broken connection` are common and normal and do not indicate log loss. See [Case Studies: Rate of Network Errors](#rate-of-network-errors) for guidance on acceptable network error rate frequencies.
```

Comment:
nit: I'd say common or typical, not acceptable

Reply:
Changed